lib/simplecov/filter.rb

```diff
@@ -25,7 +25,14 @@ def passes?(source_file)
       matches?(source_file)
     end
 
+    def self.build_filter(filter_argument)
+      return filter_argument if filter_argument.is_a?(SimpleCov::Filter)
+      class_for_argument(filter_argument).new(filter_argument)
+    end
+
     def self.class_for_argument(filter_argument)
+      return filter_argument if filter_argument.is_a?(SimpleCov::Filter)
```

Comment:
do we need this? I thought if needed this should be handled by `.build_filter` ?